psydac/fem/basic.py

```diff
@@ -56,21 +56,53 @@ def mapping( self ):
     @abstractmethod
     def vector_space( self ):
         """Topologically associated vector space."""
+        # question [MCP 03.2025]:
+            # rename as coeff_space ?
+            # meaning of docstring? see different one in MultipatchFemSpace
```

Comment:
I agree, we should rename this property! `coeff_space` sounds good to me!

Reply:
👍  in this PR?

Reply:
Ok I'm doing this here as well -- that's a lot of changes, but very simple (renaming)

Reply:
Ahah, actually I need to be careful: well in some cases `vector_space` actually denotes a vector-valued FemSpace... 🙄

Reply:
There were a lot of changes so I put them in a separate branch that can be merged after this PR

Reply:
Sounds good to me.